academy/3-ibc/1-what-is-ibc.md

```diff
@@ -124,18 +141,19 @@ In IBC, blockchains do not directly pass messages to each other over the network
 
 <HighlightBox type="docs">
 
-* The paths that all IBC implementations must support for committing IBC messages are defined in the [ICS-24 host state machine requirements](https://github.com/cosmos/ibc/tree/master/spec/core/ics-024-host-requirements).
-* The proof format that all implementations must produce and verify is defined in the [ICS-23 implementation documentation from Confio](https://github.com/cosmos/ibc/blob/master/spec/core/ics-023-vector-commitments/README.md).
+- The paths that all IBC implementations must support for committing IBC messages are defined in the [ICS-24 host state machine requirements](https://github.com/cosmos/ibc/tree/master/spec/core/ics-024-host-requirements).
+- The proof format that all implementations must produce and verify is defined in the [ICS-23 implementation documentation](https://github.com/cosmos/ibc/blob/master/spec/core/ics-023-vector-commitments/README.md).
+<!-- Is this still up-to-date after dragonberry? -->
 
```

Comment:
Are these taken from ibc-go docs? The proof format comment can be removed (it isn't useful/correct information). I think the ics23 spec itself isn't the easy to obtain information from

Light clients may decide on their own proof format (solo machine for example uses signatures)

Reply:
I checked and this was indeed taken from [ibc-go docs](https://ibc.cosmos.network/main/ibc/overview.html#proofs-and-paths). Will create an issue in the ibc-go repo for it